homeassistant/components/mitemp_bt/strings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "issues": {
+    "replaced": {
+      "title": "The Xiaomi Mijia BLE Temperature and Humidity Sensor integration has been replaced",
+      "description": "The Xiaomi Mijia BLE Temperature and Humidity Sensor integration stopped working in Home Assistant 2022.7 and replaced by the Xiaomi BLE integration in the 2022.8 release.\n\nThere is no migration path possible, therefore, you have to add your Xiaomi Mijia BLE device using the new integration manually.\n\nYour existing Xiaomi Mijia BLE Temperature and Humidity sensor YAML configuration is no longer used by Home Assistant. Remove the YAML configuration from your configuration.yaml file and restart Home Assistant to fix this issue."
```

Comment:
and was replaced by the Xiaomi BLE integration

Reply:
#75779